shared-bindings/gifio/OnDiskGif.c

```diff
@@ -69,6 +71,32 @@
 //|           # minus the overhead measured to advance between frames.
 //|           time.sleep(max(0, next_delay - overhead))
 //|           next_delay = odg.next_frame()
+//|
+//|     The displayio Group and TileGrid layers can be bypassed and the image can
+//|     be directly blitted to the full screen. This can give a speed-up of ~4x to
+//|     ~6x depending on the GIF and display. This requires an LCD that uses
+//|     standard codes to set the update area, and which accepts RGB565_SWAPPED
+//|     pixel data directly:
+//|
+//|     .. code-block:: Python
+//|       # Initial set-up the same as above
+//|
+//|       # Take over display to drive directly
+//|       display.auto_refresh = False
+//|       bus = display.bus
+//|
+//|       # Display repeatedly & directly.
+//|       while True:
+//|           # Sleep for the frame delay specified by the GIF,
+//|           # minus the overhead measured to advance between frames.
+//|           time.sleep(max(0, next_delay - overhead))
+//|           next_delay = odg.next_frame()
+//|
+//|           display_bus.send(42, struct.pack(">hh", 0, odg.bitmap.width - 1))
+//|           display_bus.send(43, struct.pack(">hh", 0, odg.bitmap.height - 1))
+//|           display_bus.send(44, d.bitmap)
+//|
+//|           d.next_frame()
```

Comment:
You call `next_frame()` on line 93 too.

Reply:
thanks for both comments -- fixed now, I think?